src/OpenTK.Windowing.GraphicsLibraryFramework/Input/KeyboardState.cs

```diff
@@ -34,6 +36,15 @@ internal KeyboardState()
         {
         }
 
+        static KeyboardState()
+        {
+	        // All this really does is copy the clumsy Array of objects
+	        // with no indexer into a strongly typed array with an indexer.
+	        Array enumValues = Enum.GetValues(typeof(Keys));
```

Comment:
This can actually be cast to a `Keys[]`, if I recall correctly.

Reply:
Done